authors/bruno-skvorc/polkadot/en_US/final.md

```diff
@@ -0,0 +1,268 @@
+
+In other words:
+
+- Ethereum shards: identical (homogenous) chains with shared security, splitting the network's load by evenly distributing it
+- Parachains: different (heterogenous) chains with shared security, splitting the network's load by making sure each chain deals with its own specific context
+
+### Parathreads
+
+A special case are Polkadot's **parathreads**. Logically, these are almost identical (see ICMP section below) to parachains, but economically they differ in that they don't lease a parachain slot long term. Because the number of DOTs required to register a parathread slot is far lower than the amount one would need to register a parachain through an auction, parathreads get to enjoy the shared security of the relay chain at a much lower cost. However, they have to bid for block production (in non-refundable DOTs) at desired intervals along with other parathreads in the same parachain slot. _Parathreads are users of a special, shared parachain slot._
+
+When it's time for this slot to produce a block to send to the relay chain, the parathreads enter an auction for that block slot on the parachain slot. The top bids get into the relay chain, up to a block limit - currently 50 for the entire relay chain. On the surface, this seems like it would slow down some parathreads (i.e. some app specific blockchains deployed as parathreads) if there are more than 50 in the system, but this is not so - parathreads are designed to be used as "read often, write as needed" blockchains.
+
+For example, a copyright parathread which logs new copyright registrations every so often does not need to submit this information to the relay chain on every block, but only as new information is added into the thread. It can batch these writes and execute them in a single block once per day, week, or any other arbitrary period.
+
+One thing that should be noted is that if a parathread is using a local currency to reward collators, the collator must be natively aware of a DOT->local conversion rate if they are to be able to submit appropriate bids for block production in DOTs. That's an economic problem for a parathread to solve on its own.
+
+Parathreads also require deposits in DOTs, but as mentioned earlier they are _much_ smaller than the deposits needed for parachain slots, and there's no auction for slots - the capacity is estimated to be enough for everyone interested to join.
+
```

Comment:
Parathread section:
-A special case are Polkadot's parathreads. A special case of what?

forward reference to ICMP is not necessary
the word slot is used in many places: block slot, parachain slot, this slot (?) , parathread slot and sometimes the word slot is not used. It causes a bit of confusion.
what type of auction is used?
comparison to mining in Bitcoin and Ethereum
-line 40 seems redundant